.github/CONTRIBUTING.md

```diff
@@ -306,43 +306,68 @@ eg. on MacOS you can follow this instructions:
 #1 install brew (https://brew.sh)
 /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
 
-#2 install node 16x using NVM (https://github.com/nvm-sh/nvm)
-nvm install v16.17.0`
-nvm alias default v16.17.0`
+# before next step uninstall node
+brew uninstall --ignore-dependencies node 
+brew uninstall --force node
+* delete .npm and .node folders from your root folder *
 
-#3 install postgres (https://wiki.postgresql.org/wiki/Homebrew)
+#2 install nvm
+brew update 
+brew install nvm
+
+# create a directory for nvm in your root folder
+mkdir ~/.nvm 
+
+# configure env variable
+vim ~/.bash_profile
+
+# paste these lines
+export NVM_DIR=~/.nvm

```

Comment:
This is added automatically when installing nvm, (but you need to open a new tab)